cpp/src/tests/CMakeLists.txt

```diff
@@ -150,29 +150,29 @@ ConfigureTest(PAGERANK_TEST "${PAGERANK_TEST_SRC}" "")
 
 ###################################################################################################
 # - SSSP tests ------------------------------------------------------------------------------------
-set(SSSP_TEST_SRCS
```

Comment:
Same for jaccard and louvain, the GRMAT dependency is limited to a specific google test. Disabling only these tests would allow other tests to run.
For SSSP it is a feature of the test app that can be deleted.

Reply:
Yes. This is what I have done.

Reply:
I am not seeing this in the PR.

Reply:
Again, what you're saying is to selectively disable the GRMAT-using tests, _only_ for Louvain, Jaccard, SSSP. Correct? I can do so, but that's changing a lot of code (potentially) that we want to revive, anyway. This was just a temporary measure until RMAT is operational in Gunrock again. Which, in fact, already is (I'm told). It's just a matter of rewriting our GRMAT tests to use their new API (why they canhed their RMAT API...that I don't know). So, I doubt there's much value in spending time changing five tests so that we change them back in the new release. We might as well disable them for now.

Reply:
I actually would have been in favor of _not disabling_ any test, because that to me is tantamount to a regression; but, fix the RMAT first. But that was overruled in favor of a faster fix for this release, and taking care of RMAT in the next.